docs/user/faq.md

```diff
@@ -94,6 +94,12 @@ x86_64-apple-darwin13.4.0-clang: No such file or directory
 
 are a telltale sign that you are lacking compilers.
 
+Depending on the specific tools that you use for building C/C++ code, you may need some additional tools, for example:
+
+```
+compilers cmake pkg-config make ninja meson
```

Comment:
```suggestion
c-compiler cxx-compiler pkg-config cmake make ninja
```

While I appreciate that using `compilers` is a one-stop way to ensure both `c-compiler` and `cxx-compiler` get added, it feels a bit strange to also suggest that people just install `fortran-compiler` even if they may have no need for it. Similarly it seems strange to suggest installing `meson` as well.

My views here are all conditioned on a research field that is a bit more conservative about pulling in additional dependencies that are not strictly required, and that I like people to generally understand why they have most of (not going to say _all_) of the dependencies installed that they do. At the same time, I'm being slightly inconsistent by not complaining about `ninja`, but `ninja` is awesome and when it can get automatically used that's even better.

So my suggestion here is more along the lines of sticking to the

> C/C++ code

part of your additions.

Alternatively/Additionally, I'd suggest mentioning what each of these dependencies brings / why you'd want to install them.

Reply:
I agree on not installing a fortran compiler, applied your suggestions on https://github.com/conda-forge/conda-forge.github.io/pull/2374/commits/85dbd591083f232374771aa8f58cd6d2d8a45639 . As a personal opinion, I think that instead is fruitful to have `meson` here, as there are huge communities in C/C++ world that consistently use `meson` (gnome, mesa, numpy&scipy), but let's what is the consensus around this.

Reply:
I'll defer to you and other reviewers here. I come from a field where in terms of build systems CMake is considered the "new" way of doing things and we're slowly clawing our way out of `autotools` + custom stuff. I only learned that `meson` even existed when SciPy started their transition to it as their build system.